Patches/Kit's VFE Weapons/ThingDefs_Misc/Weapons.xml

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success></success>
+		<operations>
+			<li Class="PatchOperationFindMod">
+			<mods><li>Kit's VFE Weapons</li></mods>
+			<match Class="PatchOperationSequence">
+			<operations>
+			
+			<!-- ==========  Hammer  =========== -->
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="KIT_CryptoHammer"]/tools</xpath>
+				<value>
+				  <tools>
+					<li Class="CombatExtended.ToolCE">
+					  <label>handle</label>
+					  <capacities>
+						<li>Poke</li>
+					  </capacities>
+					  <power>4</power>
+					  <cooldownTime>1.84</cooldownTime>
+					  <armorPenetrationBlunt>1.125</armorPenetrationBlunt>
+					</li>
+					<li Class="CombatExtended.ToolCE">
+					  <label>head</label>
+					  <capacities>
+						<li>CE_CryptoBlunt</li>
+					  </capacities>
+					  <power>41</power>

```

Comment:
I'd tone this down to be more in line with the Powered Maul from CE Melee. At 41 damage, in the hands of a colonist with even a modicum of melee skill, it tends to just one-shot body parts.

Reply:
Lowered the damage to 36, which is the powered maul. Realistically it's still going to one shot limbs quite often, but yeah shoudl make it require a bit more melee skill/higher weapon quality.